.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: scala
+
+# These directories are cached to S3 at the end of the build
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot/
+
+jdk:
+  - oraclejdk8
+
+script:
```

Comment:
not sure when number of options becomes a burden for use, at least for the java template that should not be an option